### PR TITLE
[CRI-813] Tests improvements

### PR DESCRIPTION
**Pull request by mmiedlarz**

### CODE REVIEW

File: CreatubblesAPIClient/Sources/Network/OAuth/OAuth2InMemoryClient.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//
```
Comment (JakubJankowski) [inline CreatubblesAPIClient/Sources/Network/OAuth/OAuth2InMemoryClient.swift:1]: Missing license?

File: CreatubblesAPIClient/Sources/Network/OAuth/OAuth2KeychainClient.swift [hunk 2]
```diff
@@ -0,0 +1,45 @@
+//
```
Comment (JakubJankowski) [inline CreatubblesAPIClient/Sources/Network/OAuth/OAuth2KeychainClient.swift]: Missing license?

**Review by coolbnjmn — APPROVED** [review]

**Review by JakubJankowski — APPROVED** [review]
Looks good, just those 2 missing licenses and one more question: Are `MyGalleriesRequest`, `FavoriteGalleriesRequest` and `FeaturedGalleriesRequest` tested anywhere in integration tests? I can only see their unit tests.

File: CreatubblesAPIClient/Sources/Network/OAuth/OAuth2KeychainClient.swift (hunk 2 above)
Comment (mmiedlarz) [reply CreatubblesAPIClient/Sources/Network/OAuth/OAuth2KeychainClient.swift]: thanks! Fixed.

File: CreatubblesAPIClient/Sources/Network/OAuth/OAuth2InMemoryClient.swift (hunk 1 above)
Comment (mmiedlarz) [reply CreatubblesAPIClient/Sources/Network/OAuth/OAuth2InMemoryClient.swift:1]: thanks! Fixed.

**Pull request merged**